### ISSUE TITLE
Table with suggested FHRS matches with same postcode

### ISSUE THREAD

**Issue by gregrs-uk** [gregrs-uk/python-fhrs-osm#65]
In the table of OSM entities with invalid FHRS IDs for each district, include a column of those FHRS establishments not yet successfully matched which have the same postcode as the OSM entity. Each establishment should be a link to add the appropriate FHRS ID to the OSM entity in JOSM.

Thanks @rjw62 for the suggested method.

**Comment by jnicho02** [comment]
I'm sure you've realised, these are often where a restaurant has changed hands or closed and another opened. A useful trigger for tasks in Street Complete or similar some time.

**Comment by rjw62** [comment]
@jnicho02 Yes, invalid FHRS IDs are already included in my tool at https://osm.mathmos.net/survey/ thanks to a feed from @gregrs-uk . In my experience they seem to be about 50/50 between updated IDs for the same place (which can be fixed remotely using the FHRS data) and changes of name/use (which usually require a ground survey). There have been a couple of cases recently of authorities assigning new numbers to all their registered establishments -- for which the feature suggested above would be very helpful.

**Comment by gregrs-uk** [comment]
This could also include suggested FHRS matches for OSM nodes/ways which have a matching postcode. I've [started coding this](https://github.com/gregrs-uk/python-fhrs-osm/commit/2ad86690dd1f440e4d60adb55003b8822f4d52aa#diff-57c6d273d45b7cf6772598ef7066524b) in the `postcode-matching-table` branch, although it doesn't include FHRS mismatches yet.

Given the size of the table, it should probably go on a separate page for each district.
